src/Sylius/Component/Core/Model/ProductVariant.php

```diff
@@ -345,6 +345,19 @@ public function getImages()
     /**
      * {@inheritdoc}
      */
+    public function getImage()
+    {
+        if ($this->images->isEmpty()) {
+            $image = $this->getProduct()->getImage();
+        } else {
+            $image = $this->images->first();
```

Comment:
Yep but I would miss the fallback to the product image. Do you prefer to move the fallback logic elsewhere?